src/main/java/me/zeroeightsix/kami/module/modules/misc/MountBypass.java

```diff
@@ -0,0 +1,37 @@
+package me.zeroeightsix.kami.module.modules.misc;
+
+import me.zero.alpine.listener.EventHandler;
+import me.zero.alpine.listener.Listener;
+import me.zeroeightsix.kami.event.events.PacketEvent;
+import me.zeroeightsix.kami.module.Module;
+import net.minecraft.entity.passive.AbstractChestHorse;
+import net.minecraft.network.play.client.CPacketUseEntity;
+
+/*
+ * by ionar2
+ */
+@Module.Info(
+        name = "MountBypass",
+        category = Module.Category.MISC,
+        description = "Might allow you to mount chested animals on servers that block it",
+        alwaysListening = false
```

Comment:
`alwaysListening` isn't a requirement of Info, and it's default is false so you don't need it.